pennylane/ops/qubit/hamiltonian.py

```diff
@@ -550,3 +550,77 @@ def queue(self, context=qml.QueuingContext):
 
         context.append(self, owns=tuple(self.ops))
         return self
+
+    def sparse_matrix(self, wires=None):
+        """Sparse matrix representation of this Hamiltonian.
+
+        Args:
+            wires (Iterable): Wire labels that indicate the order of wires according to which the matrix
+             is constructed. If not provided, ``self.wires`` is used.
```

Comment:
```suggestion
            wires (Iterable): Wire labels that indicate the order of wires according to which the matrix
                is constructed. If not provided, ``self.wires`` is used.
```

Reply:
what is the use-case for passing different wires to the Hamiltonian object?

Reply:
If the Hamiltonian only acts on the first few wires, but we want a matrix for all wires to use in the expval evaluation!